sds_data_manager/utils/stackbuilder.py

```diff
@@ -149,6 +163,14 @@ def build_sds(scope: App, env: Environment, account_config: dict):
         )
         # etc
 
+    # create EFS
+    efs_stack.EFSStack(scope, "EFSStack", networking.vpc, env=env)
```

Comment:
Create this earlier and then pass the stack information to later steps instead of needing the configuration parameters to be passed in earlier.